alea/parameters.py

```diff
@@ -7,19 +7,18 @@ class Parameter:
 
     Attributes:
         name (str): The name of the parameter.
-        nominal_value (float, optional): The nominal value of the parameter.
-        fittable (bool, optional): Indicates if the parameter is fittable or always fixed.
-        ptype (str, optional): The ptype of the parameter.
-        uncertainty (float or str, optional):
-            The uncertainty of the parameter. If a string,
-            it can be evaluated as a numpy or scipy function to define non-gaussian constraints.
-        relative_uncertainty (bool, optional):
-        Indicates if the uncertainty is relative to the nominal_value.
-        blueice_anchors (list, optional): Anchors for blueice template morphing.
-        fit_limits (tuple, optional): The limits for fitting the parameter.
-        parameter_interval_bounds (tupe, optional): limits for computing confidence intervals
-        fit_guess (float, optional): The initial guess for fitting the parameter.
-        description (str, optional): A description of the parameter.
+        nominal_value (float): The nominal value of the parameter.
+        fittable (bool): Indicates if the parameter is fittable or always fixed.
+        ptype (str): The ptype of the parameter.
+        uncertainty (float or str): The uncertainty of the parameter.
+            If a string, it can be evaluated as a numpy or
+            scipy function to define non-gaussian constraints.
+        relative_uncertainty (bool): Indicates if the uncertainty is relative to the nominal_value.
+        blueice_anchors (list): Anchors for blueice template morphing.
+        fit_limits (tuple): The limits for fitting the parameter.
+        parameter_interval_bounds (tuple): Limits for computing confidence intervals
+        _fit_guess (float): The initial guess for fitting the parameter.
```

Comment:
For the user it's much more useful to write the property `fit_guess` here instead of the secret attribute I'd say.